pandas/core/_numba/kernels.py

```diff
@@ -0,0 +1,113 @@
+"""
+Numba 1D aggregation kernels that can be shared by
+* Dataframe / Series
+* groupby
+* rolling / expanding
+
+Mirrors pandas/_libs/window/aggregation.pyx
+"""
+import numba
+import numpy as np
+
+
+@numba.jit(nopython=True, nogil=True, parallel=False)
+def is_monotonic_increasing(bounds):
```

Comment:
similar questions about typing (even if it doesn't actually help perf we should do it)